Aardvark/ARKLogTableViewController.m

```diff
@@ -278,7 +278,7 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
             backgroundColor = [UIColor redColor];
             break;
         case ARKLogTypeDefault:
-            textColor = [UIColor blackColor];
+            textColor = nil;
```

Comment:
Can you add a comment here explaining what `nil` ends up doing? This isn't terribly obvious without a comment.

Reply:
Good idea - added 👍